ambry-api/src/main/java/com/github/ambry/accountstats/AccountStatsStore.java

```diff
@@ -104,6 +111,12 @@
    */
   void takeSnapshotOfAggregatedAccountStatsAndUpdateMonth(String monthValue) throws Exception;
 
+  /**
+   * Delete the snapshot of latest aggregated account stats. This is usually used to prepare for the new snapshot.
+   * @throws Exception
+   */
+  void deleteSnapshotOfAggregatedAccountStats() throws Exception;
```

Comment:
Rename it to `deleteMonthlySnapshotOfAggregatedAccountStats` ? Feel like this is more consistent with what we do at `aggregatedAccountReportsDao`

Reply:
I think stick to `deleteSnapshotOfAggregatedAccountStats` since the "write" method is `takeSnapshotOfAggregatedAccountStatsAndUpdateMonth`. If we ignore the implementation, then this interface means that we can somehow take a snapshot of current aggregated account stats and attach a "month" tag with this snapshot. We only have one snapshot, if we move to next month, the snapshot will be replaced with new data and new "month" tag. So here, we just remove the snapshot.